libs/agent-sm/agent-policy/src/main/java/org/opensearch/secure_sm/policy/PolicyEntry.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+package org.opensearch.secure_sm.policy;
+
+import java.security.CodeSource;
+import java.security.Permission;
+import java.util.ArrayList;
+import java.util.List;
+
+public class PolicyEntry {
```

Comment:
This appears to be unused. There's a nested `PolicyEntry` class in PolicyFile.java that seems to be the one in use.

Reply:
thanks. think i forgot to clean them up while playing with moving classes. 

Now, removed.